pkg/resiliency/resiliency.go

```diff
@@ -840,6 +897,11 @@ func (e *circuitBreakerInstances) Remove(name string) {
 	e.Unlock()
 }
 
+// HasRetries returns true if the policy is configured to have more than 1 retry.
+func (p PolicyDescription) HasRetries() bool {
```

Comment:
any specific reason this has to be an exported function in this PR?

Reply:
That is needed in #4903